File: testing/gen_50node_3net_ttwo_data.m
% Note, run this from the testing folder
net_def = categorical([repmat("A", 1, 25), repmat("B", 1, 15), repmat("C", 1, 10) ]);
[~, ~] = fCOuNT_generate_test_data(100, 50, net_def, [0.3, 0.2, 0.5, 0, 0, 0], [0.1, 0.3, 0.1, 0, 0, 0], "ttwo", 311, "50node_3net");

File: testing/gen_50node_3net_ttwo_data.R
# Note, run this from the testing folder
fCOuNT_generate_test_data(n=90,
                          k=50,
                          net_def=c(rep("A", 25), rep("B", 15), rep("C", 10)),
                          test_type="t.two",
                          mu=c(0.2, 0.2, 0.5, 0, 0, 0),
                          tau=c(0.1, 0.3, 0.1, 0, 0, 0),
                          seed=123) %>%
  saveRDS("testdata_50node_3net_ttwo.RDS")
